config/config.exs

```diff
@@ -5,6 +5,11 @@
 # is restricted to this project.
 use Mix.Config
 
+config :sing_for_needs, SingForNeeds.Repo,
+  database: "sing_for_needs_repo",
+  username: System.get_env("POSTGRES_USERNAME") || "postgres",
+  password: System.get_env("POSTGRES_PASSWORD") || "postgres",
+  hostname: "localhost"
 # Configures the endpoint
```

Comment:
Between this and the other configs you clearly have a notion of what the default DB creds are. There are some benefits to wrapping that idea in a named abstraction, a possibility you may have already passed by and  which I remind you of only in case it should pique your interest. I know plenty of devs who would call that excessive overhead.

You also have a clear notion of a database base name. Similar tradeoffs apply.